instrumentation/opentelemetry-instrumentation-botocore/src/opentelemetry/instrumentation/botocore/extensions/types.py

```diff
@@ -0,0 +1,45 @@
+from typing import Any, Dict
+
+from opentelemetry.trace import SpanKind
+
+_BotoClientT = "botocore.client.BaseClient"
+
+_OperationParamsT = Dict[str, Any]
+
+
+class _AwsSdkCallContext:
```

Comment:
could this be a named tuple instead?

Reply:
there might be some extensions later on (e.g. SQS) that want to return a different `span_kind` or `span_name`, so for that they would be required to update the fields on this object.